File: test/fixtures/java-unirest/get_cookie_with_empty_value.java
import kong.unirest.HttpResponse;
import kong.unirest.Unirest;

class Main {
    public static void main(String[] args) {
            HttpResponse<String> httpResponse
               = Unirest.get("http://localhost:28139/cookies")
            .header("accept", "application/json")
            .header("Cookie", "mysamplecookie=someValue; emptycookie=; otherCookie=2")
            .asString();
    }
}

File: test/fixtures/kotlin-ktor/get_cookie_with_empty_value.kt
import io.ktor.client.HttpClient
import io.ktor.client.request.get
import io.ktor.client.request.header
import io.ktor.client.statement.bodyAsText
import io.ktor.http.isSuccess
import kotlinx.coroutines.runBlocking

val client = HttpClient()

runBlocking {
    val response = client.get("http://localhost:28139/cookies") {
        header("accept", "application/json")
        header("Cookie", "mysamplecookie=someValue; emptycookie=; otherCookie=2")
    }
    if (!response.status.isSuccess()) throw Exception("Request failed with status code ${response.status}")

    println(response.bodyAsText())
}
